cli/js/40_test.js

```diff
@@ -101,7 +105,15 @@ function assertExit(fn, isTest) {
 
     try {
       const innerResult = await fn(...new SafeArrayIterator(params));
-      if (innerResult) return innerResult;
+      const exitCode = DenoNs.exitCode;
+      if (exitCode !== 0) {
+        throw new Error(
+          `${isTest ? "Test case" : "Bench"} set the exit code to ${exitCode}.`,
```

Comment:
Maybe something like "Test case finished with exit code set to ${exitCode}." to imply that someone can set this to `0` at the end of their test?